bundles/org.openhab.binding.dmx/README.md

```diff
@@ -227,6 +227,11 @@ dmxActions.sendFade("1:41/3","10000:255,255,255:-1", false)
 The parameters are the same as in a chaser thing configuration.
 Defining more than one step in `fadeString` is supported, too.
 
+Example for ECMAScript
```

Comment:
Maybe you can add tabs for it instead?

See the Omnilink Binding: 
* https://next.openhab.org/addons/bindings/omnilink/#rule-actions
* https://github.com/openhab/openhab-addons/blame/main/bundles/org.openhab.binding.omnilink/README.md#L151-L169